backend/Tasque.Core.Common/DTO/Task/TaskStateEditDto.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Tasque.Core.Common.DTO.Task
+{
+    public class TaskStateEditDto
+    {
+        public string Name { get; set; } = null!;
```

Comment:
Task state also has color

Reply:
I actually don't know what data was needed there, I just created a new DTO so whoever is working on it can extend it if needed